### DISCUSSION TITLE
python import

### DISCUSSION THREAD

**Discussion by janheuser** [blaylockbk/Herbie#382] [Q&A]
This seems very silly...but I am having issues importing herbie into a notebook and can't seem to find this discussed elsewhere here. The notebook is on an AWS Cloud9 instance running AWS Linux. I can see that conda has access to herbie-data and even created the herbie venv in case that was the issue. It seems to me that whatever aliasing connects "import herbie" to the package titled herbie-data isn't working. Any thoughts as to why?

**Comment by Cberg93** [comment]
Yea no idea

**Comment by loveinabox** [comment]
maybe you can try to check environment variables: Since you mentioned using the virtual environment created with conda, make sure you activate the correct conda environment before trying to import herbie. You can activate the environment by running conda activate your env name, where your env name is your environment name.

if still cant, you can also try to check aliases and paths: You mention the "aliasing" problem, which may mean that the Python interpreter does not correctly point import herbie to the herbie-data package. Check your PYTHONPATH environment variable to make sure it contains the path to the herbie-data package.
